core/client/fs/src/test/java/alluxio/client/file/FileSystemTestUtils.java

```diff
@@ -107,9 +107,8 @@ public static void createByteFile(FileSystem fs, AlluxioURI fileURI, CreateFileP
    */
   public static void createByteFile(FileSystem fs, String fileName, WritePType writeType, int len,
       long blockCapacityByte) {
-    CreateFilePOptions options = FileSystemClientOptions.getCreateFileOptions().toBuilder()
-        .setWriteType(writeType).setBlockSizeBytes(blockCapacityByte)
-        .build();
+    CreateFilePOptions options = CreateFilePOptions.newBuilder().setWriteType(writeType)
+        .setBlockSizeBytes(blockCapacityByte).setRecursive(true).build();
```

Comment:
why is recursive set now?

Reply:
Alluxio Java API change. Explained in other comment.